examples/typescript/provisioner/cdktf.json

```
@@ -0,0 +1,8 @@
+{
+  "language": "typescript",
+  "app": "npx ts-node main.ts",
+  "terraformProviders": [
+    "kreuzwerker/docker@~> 2.0",
+    "null"
```

nit: adding a version to the null provider might protect from future failures due to incompatible changes
```suggestion
    "null@~> 3.1"
```